upgrading from 3.4 single container to 3.6 single container fails

Migrated from https://pulp.plan.io/issues/7451.

The logs for the single container display the following traceback. This is due to pulp storing symlinks in /var/lib/pulp/assets/. The symlinks for drf-yasg are now broken because pulpcore no longer uses that package.

```
[2020-09-03 16:26:25 +0000] [13378] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.7/site-packages/gunicorn/workers/base.py", line 119, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.7/site-packages/gunicorn/workers/base.py", line 144, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 49, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 39, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.7/site-packages/gunicorn/util.py", line 358, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib64/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/local/lib/python3.7/site-packages/pulpcore/app/wsgi.py", line 17, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python3.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    return WSGIHandler()
  File "/usr/local/lib/python3.7/site-packages/django/core/handlers/wsgi.py", line 135, in __init__
    self.load_middleware()
  File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py", line 37, in load_middleware
    mw_instance = middleware(handler)
  File "/usr/local/lib/python3.7/site-packages/whitenoise/middleware.py", line 50, in __init__
    self.add_files(self.static_root, prefix=self.static_prefix)
  File "/usr/local/lib/python3.7/site-packages/whitenoise/base.py", line 113, in add_files
    self.update_files_dictionary(root, prefix)
  File "/usr/local/lib/python3.7/site-packages/whitenoise/base.py", line 120, in update_files_dictionary
    stat_cache = dict(scantree(root))
  File "/usr/local/lib/python3.7/site-packages/whitenoise/base.py", line 271, in scantree
    yield from scantree(entry.path)
  File "/usr/local/lib/python3.7/site-packages/whitenoise/base.py", line 273, in scantree
    yield entry.path, entry.stat()
FileNotFoundError: [Errno 2] No such file or directory: '/var/lib/pulp/assets/drf-yasg/immutable.min.js'
```

This issue can be worked around by removing the 'assets/drf-yasg' directory from the pulp storage directory on the host machien.

--- a/assets/pulpcore-api.run
+++ b/assets/pulpcore-api.run
@@ -2,7 +2,7 @@
 foreground {
   export DJANGO_SETTINGS_MODULE pulpcore.app.settings
   export PULP_CONTENT_ORIGIN localhost
-  /usr/local/bin/pulpcore-manager collectstatic --noinput --link
+  /usr/local/bin/pulpcore-manager collectstatic --clear --noinput --link
 }
 export DJANGO_SETTINGS_MODULE pulpcore.app.settings
 export PULP_SETTINGS /etc/pulp/settings.py

